e2e/framework/addon/conjur.go

```diff
@@ -42,33 +46,46 @@ type Conjur struct {
 
 	AdminApiKey    string
 	ConjurServerCA []byte
+	portForwarder  *PortForward
 }
 
-func NewConjur(namespace string) *Conjur {
-	repo := "conjur-" + namespace
+func NewConjur() *Conjur {
```

Comment:
whats the reason to hardcode the namespace here?

Reply:
We don't need it to be dynamic. both conjur and vault are installed/configured once in a single place